EventPlaning.BL.Tests/EventManagerTests.cs

```diff
@@ -110,7 +122,9 @@ public void RemoveActivityFail()
                 Title = "Monopoly",
             };
 
-            Assert.IsFalse(eventManager.RemoveActivityFromEvent(2, activity));
+            eventManager.RemoveActivityFromEvent(2, activity);
+
+            Assert.AreEqual("Monopoly", eventManager.GetEventActivities(1)[1].Title);
```

Comment:
If you change initial data, the same test will fail (because the new event activity has changed position in result of the method)